telegram/_bot.py

```diff
@@ -2037,16 +2051,40 @@ async def send_media_group(
                 :attr:`~telegram.request.BaseRequest.DEFAULT_NONE`.
             api_kwargs (:obj:`dict`, optional): Arbitrary keyword arguments to be passed to the
                 Telegram API.
+            group_caption (:obj:`str`, optional): Text of the caption
+                that will be shown for the whole media group.
```

Comment:
Maybe "the caption is added to the first element of :paramref:`media`, such that it will be used as caption for the whole media group"?

Reply:
yes, fixing